indico/modules/users/models/users.py

```diff
@@ -44,16 +44,17 @@ class UserTitle(RichIntEnum):
 
 
 class NameFormat(RichIntEnum):
-    __titles__ = (_('John Doe'), _('Doe, John'), _('Doe, J.'), _('J. Doe'),
+    __titles__ = (_('Inherit from user settings'), _('John Doe'), _('Doe, John'), _('Doe, J.'), _('J. Doe'),
                   _('John DOE'), _('DOE, John'), _('DOE, J.'), _('J. DOE'))
-    first_last = 0
-    last_first = 1
-    last_f = 2
-    f_last = 3
-    first_last_upper = 4
-    last_first_upper = 5
-    last_f_upper = 6
-    f_last_upper = 7
+    inherit = 0
```

Comment:
unless you add an alembic revision to update all existing name form at user settings you can't just change those values since they are stored in the database...

Maybe just use `-1` for inherit? Or like i initially suggested, don't change the enum at all but instead use `None` on the event-level name format to indicate inheritance